src/main/java/com/desiato/puresynth/exceptions/CustomAuthenticationException.java

```diff
@@ -0,0 +1,7 @@
+package com.desiato.puresynth.exceptions;
+
+public class CustomAuthenticationException extends RuntimeException {
```

Comment:
Why do you need it? Isn't it 3rd auth exception? What's the difference between eception from Spring, this and InvalidTokenException?

Reply:
I do not need CustomAuthenticationException because it does not add value beyond what Spring's exception and InvalidTokenException already do. So I removed it.
I initially added it when I created GlobalExceptionHandler, because I thought I needed this additional custom exception, but I was wrong.